tests/glsl-intrinsic/raytracing/glsl-rayAnyhit-cluster.slang

```diff
@@ -0,0 +1,34 @@
+//TEST:SIMPLE(filecheck=CHECK_GLSL): -allow-glsl -stage anyhit -entry main -target glsl -capability GL_EXT_ray_tracing -capability GL_NV_cluster_acceleration_structure
```

Comment:
This test is really huge, can it be made smaller to test just the essentials, if something starts failing here it's gonna be a task to minimise it